src/select/Select.ts

```diff
@@ -250,7 +250,7 @@ export default class Select extends SelectBase<SelectProperties> {
 		});
 	}
 
-	renderNativeSelect(): DNode {
+	protected renderNativeSelect(): DNode {
```

Comment:
I think it was already visible, just missing the `protected` keyword. The other similar fn's have it, so I thought I'd sneak it in :P